webapp-django/crashstats/crashstats/views.py

```diff
@@ -2019,7 +2019,8 @@ def format_float(number):
             'product': r['product_name'],
             'version': r['version_string'],
             'percentage': format_float(r['percentage']),
-            'numberOfCrashes': r['report_count']})
+            'numberOfCrashes': r['report_count'],
+            'rank': r.get('currentRank', '')})
```

Comment:
I can't find a single place where this is ever present. I clicked on a bunch of signatures under Firefox 41 and it's never there. I think I need to dig deeper to try to understand what you tried achieve.